### DISCUSSION TITLE
How to calculate the radius of bubble in the MOOSE?

### DISCUSSION THREAD

**Discussion by wenpeng231024** [idaholab/moose#18379] [Q&A General] [answered]
Dear MOOSE group,

I try to use Laplace's equation (2γ/R) to calculate the pressure produced by the surface tension. γ is the surface tension. R is the radius of the bubble.

Surface tension can be obtained using "ComputeSurfaceTensionKKS".

For a single bubble, we can use the order parameters to obtain the bubble area, then calculate the radius of the bubble.

But how can we calculate different radii of multi-bubbles in the MOOSE? Or how can we calculate the curvature of the bubbles?

Thanks,

Peng

**Comment by SudiptaBiswas** [comment, marked as answer]
You can use `FeatureVolumeVectorPostprocessor` to output the volume/area of the bubbles. https://mooseframework.inl.gov/source/vectorpostprocessors/FeatureVolumeVectorPostprocessor.html

**Reply by wenpeng231024** [reply]
Great. I will try this. Many thanks!

Peng
